### DISCUSSION TITLE
Use pin D8 for something different?

### DISCUSSION THREAD

**Discussion by selmaohneh** [adonno/tagreader#196] [Q&A]
Hi! 

I do not use the LED but instead want to add a push button to the GPIO D8. What do I need to do to remove the LED / free the PIN? Is there a way to configure that? Or do I have to somehow edit the firmware? :-/ All other PINs are already used, as well. D0 is free but does not have a pullup resistor - so I really need to use D8.

Thanks in advance for any help!

**Comment by adonno** [comment]
Did you add tagreader to ESPHome ?

You could copy the yaml form this repo and get rid of the LED parts
